pkg/reconciler/apiserversource/apiserversource_test.go

```diff
@@ -657,8 +660,13 @@ func TestReconcile(t *testing.T) {
 		SkipNamespaceValidation: true, // SubjectAccessReview objects are cluster-scoped.
 	}}
 
+	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(
+		fakekubeclientset.NewSimpleClientset(), 0)
+	svcInformerIface := kubeInformerFactory.Core().V1().Services()
```

Comment:
If this interface isn't actually used in tests, is it necessary to explicitly request this informer, as opposed to just using a magic `_` injection import?

Reply:
80c930f